10_code/tools/tagging_tool.py

```diff
@@ -0,0 +1,128 @@
+import streamlit as st
+import os
+import pandas as pd
+
```

Comment:
Do you mind me trying to add keyboard shortcuts?

Reply:
sure, go ahead.